.github/workflows/deploy-apis-to-staging.yml

```diff
@@ -162,6 +164,10 @@ jobs:
               echo "::set-output name=run_airflow::true"
             fi
 
+            if [[ $file == src/airflow/* ]]; then
+              echo "::set-output name=run_network_uptime::true"
+            fi
+
```

Comment:
@123MwanjeMike there is a typo here

Reply:
Thanks @NoahNsimbe. Let me fix these typos asap